UPGRADE.md

```diff
@@ -0,0 +1,259 @@
+# GraphDB Helm Chart Upgrade Guide
+
+## From 10.x to 11
+
+Before continuing, familiarize yourself with the new configuration structure in [values.yaml](values.yaml) as well as with
+the [changelog for version 11](CHANGELOG.md#version-1100).
+
+### Configurations
+
+Version 11 of the Helm chart introduces a lot of changes to the configurations and their structure in [values.yaml](values.yaml). 
+Here are the most notable steps that you should be aware of:
+
+**Kubernetes Version**
+
+The Helm chart is updated to require a minimum Kubernetes version of 1.26.
+Refer to the official Kubernetes documentation on how to move to a newer version if needed.
+We strongly advise to operate a Kubernetes cluster with version that has not reached its End Of Life. 
+
+**Naming**
+
+The resource names are no longer hardcoded in version 11. If you want to keep the old ones, you can override the following configurations from 
+[values.yaml](values.yaml): `fullnameOverride` and `proxy.fullnameOverride`.
+
+**GraphDB URL**
+
+The old version 11 of the chart had several configuration properties that defined the external URL of GraphDB: `deployment.protocol`, 
+`deployment.host` and `graphdb.workbench.subpath`.
+In version 11, they have been combined into a single configuration property `configuration.externalUrl`.
+Make sure to update it accordingly.
+
+Note that the default URL configuration now uses https://nip.io/ which avoids having to edit your hosts file, compared to version 10.
+However, this is suitable only for local development, so you'd have to update it to a real resolvable URL according to your environment and
+requirements.
+
+**Ingress**
+
+If you use the Ingress for accessing GraphDB: Version 11 removes the default use of specific ingress controllers.
+It's up to you to properly set up the default ingress controller in your cluster or to properly configure the default Ingress in this chart with
+the `ingress` configuration in [values.yaml](values.yaml).
+
+**Storage Class**
+
+Version 11 removes the hardcoded storage class from `global.storageClass`.
+If you don't have a default storage class in your cluster, you can define the storage class for GraphDB's PVC
+with `persistence.volumeClaimTemplate.spec.storageClassName` for GraphDB and `persistence.volumeClaimTemplate.spec.storageClassName` for the proxy (if
+enabled).
+
+**Persistence**
+
+Persistence configurations have been moved to `persistence` and `proxy.persistence`.
```

Comment:
This is a bit confusing, all persistence configurations?

Reply:
I'll mention the previous location